src/app/page.tsx

```diff
@@ -1,19 +1,46 @@
 "use server";
 
-import { ColumnRepository } from "@/model/Column";
-import { TaskRepository } from "@/model/Task";
 import { User } from "@/model/types";
 import { Home } from "@/templates/Home";
 import { cookies } from "next/headers";
 
-const taskRepository = new TaskRepository();
+const fetcher = (query: string) =>
```

Comment:
This is being read server-side. You can just call the Repositories directly without GraphQL calls.